src/api/migrations/20190703212525-likeDislike.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('LikeDislikes', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    commentId: {
+      allowNull: false,
+      type: Sequelize.INTEGER,
+      references: { model: 'Comments', key: 'id' }
+    },
+    userId: {
```

Comment:
Do we have a way of seeing the names of the people who liked the comment? @murediane

Reply:
I don't think we need to, Because I don't think the main idea is to know who liked but how people, in general, feel about a comment.